src/worker.ts

```diff
@@ -59,6 +59,9 @@ registerPromiseWorker(async (message: PostMessageArgs): Promise<any> => {
       case "getResponseCacheSize":
         result = await client.getResponseCacheSize();
         break;
+      case "getSchemaTypeMap":
+        result = await client.getSchemaTypeMap();
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Invalid 'await' of a non-Promise value. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=typescript%3AS4123)